ci/dev_menu.py

```diff
@@ -148,7 +148,7 @@ def provision_virtualenv(venv_path=DEFAULT_PYENV):
     ]),
     ('[Docker] Python3 CPU Intel MKLDNN unittests',
     [
-        "ci/build.py --platform ubuntu_cpu /work/runtime_functions.sh build_ubuntu_cpu_mkldnn",
+        "ci/build.py --platform ubuntu_cpu /work/runtime_functions.sh build_ubuntu_cpu_onednn",
         "ci/build.py --platform ubuntu_cpu /work/runtime_functions.sh unittest_ubuntu_python3_cpu",
     ]),
     ('[Docker] Python3 ARMv7 unittests (QEMU)',
```

Comment:
@leezu I saw that some of the functions called here are not present in ci/docker/runtime_functions.sh file. Is there a separate file for CI stored outside of this repository or maybe these names should be updated?

Reply:
This file is outdated and needs to be updated